bundles/io/org.eclipse.smarthome.io.json/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,7 @@
+eclipse.preferences.version=1
```

Comment:
Please remove this preferences file, projects should use the workspace default settings.

Reply:
Removed, I think it was generated by Eclipse's "new project wizard"